modules/social_features/social_landing_page/templates/group--featured.html.twig

```diff
@@ -95,7 +95,9 @@
     </span>
   {% endif %}
 
-  <a href="{{ url }}" class="card__link" title="{{ label }}">
-    {% trans %}Read more{% endtrans %}
-  </a>
+  {% if not hide_card_link %}
```

Comment:
@OH17L Have you tested if this condition will work correctly without Dashboard module? I mean if it will be always displayed if there is no variable set?

Reply:
@ribel, yes, I tested it and it works correctly